src/main/scala/chisel3/Driver.scala

```diff
@@ -149,7 +149,11 @@ object Driver extends BackendCompilationUtilities {
     /* create custom transforms by finding the set of transform classes associated with annotations
      * then instantiate them into actual transforms
      */
-    val transforms = circuit.annotations.map(_.transform).toSet.map { transformClass: Class[_ <: Transform] =>
+    val transforms = circuit.annotations
+                            .map(_.transform)
+                            .distinct
+                            .filterNot(_ == classOf[firrtl.Transform])
```

Comment:
This whole chunk of code instantiating transforms from annotations probably should be moved to Firrtl and we should rethink how we decide what transforms to instantiate (since it's possible someone will annotate a transform that can't be instantiated in this way). 

Specifically, this filterNot is preventing annotations that target firrtl.Transform (like dont touch) don't cause it to try to instantiate an instance of firrtl.Transform since it is an abstract class without a constructor. 

So long as this code stays in chisel this workaround is kind of mandatory because without it we get java invocation target exceptions when we try to have dont touch annotations.